app/assets/stylesheets/application.css

```
@@ -13,3 +13,326 @@
  *= require_tree .
  *= require_self
  */
+ 
+
+:root {
+  --main-bg-color: #036635;
+}
+
+
+body {
+background-image: url('star.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-attachment: fixed;
+}
+
+.background-white {
+  background-color: rgba(255, 255, 255, 0.8);
+  padding: 30px 10px;
+}
+
+.wf-nicomoji { font-family: "Nico Moji"; }
+
+
+/*ヘッダー*/
+header {
+  height: 64px;
+  width: 100%;
+  background-color: rgba(3, 102, 53, 0.8);
+}
+
+.header-logo {
+  float: left;
+  padding-left: 20px;
+  color: white;
+  font-size: 22px;
+  line-height: 64px;
+}
+
+.header-logo a{
+  color: white;
+  font-size: 22px;
+}
+
+.header-menus {
+  float: right;
+  padding-right: 20px;
+}
+
+.header-menus li {
+  float: left;
+  line-height: 64px;
+  font-size: 13px;
+  color: white;
+  padding-left: 15px;
+}
+
+.header-menus a {
+  float: left;
+  font-size: 13px;
+  color: white;
+}
+
+@media screen and (max-width: 480px){
+  
+  .header-logo a{
+    font-size: 18px;
+  }
+  
+  .header-menus {
+    padiing-right: 10px;
+  }
+  
+  .header-menus li {
+    padding-left: 8px;
+  }
+  .header-menus a {
+    font-size: 8px;
+  }
+}
+
+.header-menus .fa {
+  padding-right: 5px;
+}
+
+.header-menus input[type="submit"] {
+  padding: 0 20px;
+  float: left;
+  line-height: 64px;
+  color: white;
+  margin: 0;
+  font-size: 13px;
+}
+
+
+
+/* login sign_up*/
+.my-div {
```

そもそもクラス名を分かりやすくすれば、コメントは不要そう

```suggestion
.login, .sign_up {
```